public/stylesheets/detailsPage.css

```diff
@@ -0,0 +1,63 @@
+.category-color {
+  background: #F5DECD;
+}
+
+#category-bg {
+  position: relative;
+  width: 100%;
+  height: auto;
+  height: 200px;
+}
+
+.app-logo {
+  width: 200px;
+  height: 200px;
+  background-size: cover;
+  border-radius: 8px;
+  position: absolute;
+  bottom: -100px;
+  left: 0;
+  right: 0;
+  margin: auto;
+}
+
+.app-title {
+  margin-top: 140px;
+  text-align: center;
+}
+
+.description {
+  padding: .5rem!important; 
+}
+
+.btn {
+  color: black;
+  border-color: black;
+  box-shadow: none;
+}
+
+.mr-3 {
+  margin-top: 130px;
+}
```

Comment:
better to have a custom class to change this. mr-3 or all style from bootstrap should have its original styles, if we have a class and you want to overwrite bootstrap class, you can add !important

Reply:
Added it to the new id details-btn.